moxy/eclipselink.moxy.test/pom.xml

```diff
@@ -143,7 +143,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.20.1</version>
+                <version>2.21.0</version>
```

Comment:
I remember that there were issues in some metro project with 2.21.0. But if it works here, its good.